doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -74,8 +75,21 @@ lose their role as parameters. For example:
 
    Notation "'IF' c1 'then' c2 'else' c3" := (c1 /\ c2 \/ ~ c1 /\ c3) (at level 200, right associativity).
 
-Symbols that start with a single quote with 3 or more characters must be single quoted.
-For example, the symbol `'ab` is represented by `''ab'` in the notation string.
+Symbols that start with a single quote and followed by at least 2
+characters must be single quoted.  For example, the symbol `'ab` is
+represented by `''ab'` in the notation string. Strings can be used in
+notations: the start and end of a string symbol in a notation string
```

Comment:
```suggestion
notations: the start and end of a string token in a notation string
```

Perhaps clearer if you refer to tokens instead of symbols?  If so, probably need to change elsewhere, too.

Reply:
I hesitated also. "Token" looks more natural to use but it is less precise. On the other way, a string token can only be a symbol.

Reply:
Oh?  Characters are symbols, too, so `"abc"` has 5 symbols.  "Token" is the precise term in parsing IMO.  To me, "symbol" suggests something that is a non-terminal (not a literal).

Reply:
Maybe worth to follow the common usage then, that is to use "literal" instead of "symbol" in the chapter (see paragraph "The notation consists of *tokens* separated by spaces. ... The other elements of the string (such as ``/\``) are the *symbols*")...